ground/gcs/share/pfd/default/PfdAttitudeLine.qml

```diff
@@ -0,0 +1,36 @@
+import QtQuick 2.0
+
+Item {
+    id: realAttitude
+
+    Rectangle {
+        id: realAtt
+        smooth: true
+
+        property variant scaledBounds: svgRenderer.scaledElementBounds("pfd.svg", "world")
+        width:  Math.round(sceneItem.width *scaledBounds.width /2)*2
+        height: Math.round(sceneItem.height*scaledBounds.height/2)*2
+
+        visible: AttitudeSimulated.isPresent && (AttitudeSimulated.Roll !== 0 || AttitudeSimulated.Pitch !== 0)
+
+        gradient: Gradient {
+            GradientStop { position: 0.496;    color: "#00000000" }
+            GradientStop { position: 0.500;    color: "#c0008000" }
+            GradientStop { position: 0.504;    color: "#00000000" }
+        }
+
+        transform: [
+            Translate {
+                id: pitchTranslateReallAtt
```

Comment:
Typo etc. But this doesn't actually need an id/I can't see any references to it?